flask_caching/_compat.py

```diff
@@ -12,52 +12,28 @@
 """
 import sys
 
-PY2 = sys.version_info[0] == 2
 PYPY = hasattr(sys, "pypy_translation_info")
 
 
-if not PY2:
-    range_type = range
-    iterkeys = lambda d: iter(d.keys())
-    itervalues = lambda d: iter(d.values())
-    iteritems = lambda d: iter(d.items())
-
-    text_type = str
-    string_types = (str, )
-    integer_types = (int, )
-
-    def to_bytes(x, charset=sys.getdefaultencoding(), errors='strict'):
-        if x is None:
-            return None
-        if isinstance(x, (bytes, bytearray, memoryview)):  # noqa
-            return bytes(x)
-        if isinstance(x, str):
-            return x.encode(charset, errors)
-        raise TypeError('Expected bytes')
-
-    def to_native(x, charset=sys.getdefaultencoding(), errors='strict'):
-        if x is None or isinstance(x, str):
-            return x
-        return x.decode(charset, errors)
-else:
-    range_type = xrange
-    iterkeys = lambda d: d.iterkeys()
-    itervalues = lambda d: d.itervalues()
-    iteritems = lambda d: d.iteritems()
-    text_type = unicode
-    string_types = (str, unicode)
-    integer_types = (int, long)
-
-    def to_bytes(x, charset=sys.getdefaultencoding(), errors='strict'):
-        if x is None:
-            return None
-        if isinstance(x, (bytes, bytearray, buffer)):
-            return bytes(x)
-        if isinstance(x, unicode):
-            return x.encode(charset, errors)
-        raise TypeError('Expected bytes')
-
-    def to_native(x, charset=sys.getdefaultencoding(), errors='strict'):
-        if x is None or isinstance(x, str):
-            return x
+range_type = range
+iterkeys = lambda d: iter(d.keys())
+itervalues = lambda d: iter(d.values())
+iteritems = lambda d: iter(d.items())
+
+text_type = str
+string_types = (str, )
+integer_types = (int, )
```

Comment:
I think we should remove the lines 18-25 too.

Reply:
Yeah, i was about to ask if the compat file should stay at all. I guess iʼll just remove it, then.